src/Streamly/Internal/FS/Event/Windows.hs

```diff
@@ -472,13 +461,16 @@ closePathHandleStream =
 --
 -- /Pre-release/
 --
-watchWith :: (Config -> Config) -> NonEmpty (Array Word8) -> Stream IO Event
+watchWith :: (Config -> Config) -> NonEmpty Path -> Stream IO Event
```

Comment:
Now the semantics of Path have changed. Earlier it was UTF8 encoded array, now it is a WindowsPath (UTF16 encoded). Fix the documentation above. 

Check for any other such doc changes everyhwere, in Linux/Darwin modules as well.